src/background/actions/action-creator.ts

```diff
@@ -55,21 +55,37 @@ export class ActionCreator {
     ) {
         this.visualizationActions = actionHub.visualizationActions;
         this.previewFeaturesActions = actionHub.previewFeaturesActions;
-        this.visualizationScanResultActions = actionHub.visualizationScanResultActions;
+        this.visualizationScanResultActions =
+            actionHub.visualizationScanResultActions;
         this.inspectActions = actionHub.inspectActions;
     }
 
     public registerCallbacks(): void {
-        this.interpreter.registerTypeToPayloadCallback(visualizationMessages.Common.Toggle, this.onVisualizationToggle);
-        this.interpreter.registerTypeToPayloadCallback(visualizationMessages.Common.ScanCompleted, this.onScanCompleted);
-        this.interpreter.registerTypeToPayloadCallback(visualizationMessages.Common.ScrollRequested, this.onScrollRequested);
-        this.interpreter.registerTypeToPayloadCallback(visualizationMessages.Common.RescanVisualization, this.onRescanVisualization);
+        this.interpreter.registerTypeToPayloadCallback(
```

Comment:
unneeded extra new lines

Reply:
This is just how `GitHub` is showing this. There are no extra lines here. You can also check the line numbers here or the attached screenshot from my local code instance.

![image](https://user-images.githubusercontent.com/4496335/69263962-349feb00-0b7b-11ea-833a-a8c8b0c667a8.png)